subgraph/src/fetch/scannerpool.ts

```diff
@@ -10,6 +10,7 @@ export function fetchScannerPool(id: BigInt): ScannerPool {
     scannerPool.registered = false;
     scannerPool.chainId = 1;
     scannerPool.apr = BigDecimal.zero();
+    scannerPool.oldCommission = BigDecimal.zero();
```

Comment:
Should this be 100%?

Reply:
It works anyways: it will be ignored because `commissionSinceEpoch = 0` with `100%` at the beginning. So it asumes that was 100% UI since the first epoch.

The values at the beginning are actually set in the `handleScannerPoolRegistered` function. 
```
scannerPool.commission = BigDecimal.fromString("10000");
```
I can try to add one of oldCommission here too, but it won't change anything in the UI.